File: ruby/speed.rb

def speedTest()
    start = Time.now()
    myList = []
    rand = Random.new()
    for i in 1..10000000 do
        myList.append(rand.rand(100))
    end

    # While Loop. This seems to be a bit faster than the For Loop,
    # The only language I've seen that
    # i = 0
    # while i < 100000 do
    #     i += 1
    #     myList.append(rand.rand(100))
    # end

    result = myList.sort()
    puts (((Time.now() - start).to_f * 1000)).round(4).to_s + "ms"

end

File: python/speed.py
import time
import random as rand

# Speed test in python
def speedTest():
    start = time.time()
    myList = []

    # While loop
    # i = 0
    # while i < 100000:
    #     i += 1
    #     myList.append(rand.randint(1, 100))
        
    # For loop, no visible difference in speed between this and the while loop
    for i in range(10000000):
        myList.append(rand.randint(1, 100))
        
    myList.sort()
    print(f"{round(((time.time() - start) * 1000), 4)}ms")

if "__main__" == __name__:
    speedTest()